### DISCUSSION TITLE
Show vouchers in groups in the voucher list

### DISCUSSION THREAD

**Discussion by raphaelm** [pretix/pretix#1961] [Ideas]
It would probably be better to go away from the tag concept and introduce an actual "voucher group" concept. If you create N vouchers at once, they will be placed in one group and the voucher list only shows groups, not single vouchers.

see also #145

**Comment by rixx** [comment]
How would you add vouchers to a group, then? Via drop-down?

**Comment by raphaelm** [comment]
Well, you would create a group while you create the first vouchers for that group. If you want to create more later, you can just go to the detail page of that group and say "add more"
